slsim/Sources/point_plus_extended_source.py

```diff
@@ -0,0 +1,83 @@
+import numpy.random as random
+from slsim.Sources.source_pop_base import SourcePopBase
+import numpy as np
+from slsim.selection import deflector_cut
+from slsim.Sources.galaxies import galaxy_projected_eccentricity
+
+
+class PESource(SourcePopBase):
+    """Class to describe point and extended sources."""
+
+    def __init__(
+        self,
+        pes_list,
+        cosmo,
+        sky_area,
+        kwargs_cut,
+        variability_model=None,
+        kwargs_variability_model=None,
+    ):
+        """
+
+        :param pes_list: list of dictionary with point and extended source parameters
```

Comment:
pes_list is hard to interpret as a parameter name

Reply:
Renamed it as point_plus_extended_source_list.